sqlalchemy_jsonapi/serializer.py

```diff
@@ -577,6 +577,7 @@ def delete_relationship(self, session, data, api_type, obj_id, rel_key):
                 if reverse_rel.direction == MANYTOONE:
                     permission = Permissions.EDIT
                 else:
+                    # pragma: no cover
```

Comment:
No coverage on this because I have not been able to test this, I tried for a while the different combinations with my models but I think I need to make either a 1:1 relationship so that the reverse_rel.direction won't be hit or a many to many. Either way my plan is to hold off on making these models until I see more code that requires different type of model setup.